Source/Filters/Video/YPVideoView.swift

```diff
@@ -47,12 +48,20 @@ public class YPVideoView: YPAdjustableView {
 
         updateViewFrameAction = { [weak self] frame in
             self?.playerView.layer.frame = frame
+            self?.playerView.playerLayer.speed = 999 // disable the "zoom in" animation
```

Comment:
Where did this 999 value come from? This value is just good enough for disabling the animation speed or was this a recommended value to use in this context?

Reply:
I think I just found an example from somewhere in the internet. Looks like the `speed` property is a multiplier, so a high enough value just makes it instantaneous: https://developer.apple.com/documentation/quartzcore/camediatiming/1427647-speed.